tests/importit/PathFiltersTest.php

```diff
@@ -13,6 +13,7 @@
 namespace Facebook\ImportIt;
 
 use namespace HH\Lib\{Keyset, Vec};
+use type Facebook\HackTest\DataProvider; // @oss-enable
```

Comment:
`// @oss-disable: use type DataProvider;`

Reply:
In general, hhvm/hsl is a good example (as it's synced to/from the same repo as fbshipit), but it doesn't need this as the HSL's tests aren't namespaced. Namespaced is probably better :)